pkg/utils/configMapUtils.go

```diff
@@ -116,32 +171,8 @@ func CreateConfigMaps(configMaps []v1alpha1.ConfigMap, engineDetails EngineDetai
 	return err
 }
 
-/*func CreateVolumes(configMaps []v1alpha1.ConfigMap) []corev1.Volume {
-	//volumesList := make([]corev1.Volume, len(configMaps))
-	log.Infoln("-------------------------------------")
-	log.Infoln(configMaps)
-	log.Infoln("--------------------------")
-	volumesList := []corev1.Volume{}
-
-	for _, v := range configMaps {
-		log.Infoln("RETURNING OBJECT")
-		log.Infoln(v)
-		volume := corev1.Volume{}
-		volume.Name = v.Name
-		log.Infoln("!!!!!!!!!!!!!!!!!!!!!!!!!!!! ->")
-		log.Infoln(v.Name)
-		volume.ConfigMap.Name = v.Name
-		var i int = 420
-		var k int32 = int32(i)
-		volume.ConfigMap.DefaultMode = &k
-		volumesList = append(volumesList, volume)
-		//volumesList[i] = volume
-	}
-	return volumesList
-}*/
-
 // CreateVolumeBuilder build Volume needed in execution of experiments
-func CreateVolumeBuilder(configMaps []v1alpha1.ConfigMap) []*volume.Builder {
+func CreateVolumeBuilder(configMaps []v1alpha1.ConfigMap, secrets []v1alpha1.Secret) []*volume.Builder {
```

Comment:
Do you need this secrets here??

Reply:
Yes, volume has to be built for secrets

Reply:
Maybe in future (BCH) we have separate createConfigMapBuilder and createSecretBuilder. Just thinking aloud - what if more type of volumes are to be supported.